package.json

```diff
@@ -36,14 +39,15 @@
     "qejs": "^3.0.5",
     "ractive": "^0.6.1",
     "react": "^0.14.6",
+    "react-dom": "^0.14.7",
     "react-tools": "^0.13.0",
```

Comment:
This is not true as far as I see it. The first time running this just takes _very_ long on 0.10 and 0.12 and therefor the first test fails. All tests pass if you increase the timeout to lets say 10 seconds for the first react test. It's about the NodeJS require call taking very long for the first file loaded with a react extension. I guess there are other things happening in the background in combination with react (I guess react is lazy loaded).